pkg/pop/msal.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
any unit test?

Reply:
Added some VCR tests for the confidential flow